A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -442,7 +442,7 @@ class CDUPerformancePage {
         }
         let bottomRowLabels = ["PREV", "NEXT"];
         let bottomRowCells = ["<PHASE", "PHASE>"];
-        if (mcdu.currentFlightPhase === FlightPhase.FLIGHT_PHASE_CLIMB) {
+        if (mcdu.currentFlightPhase === FlightPhase.FLIGHT_PHASE_DESCENT) {
```

Comment:
this fixes it so the "activate appr phase" action will appear on the DES page when DES phase is active